### ISSUE TITLE
error processing an exponent, bigDecimal.round(0.00000001,8) return "0.00000000"

### ISSUE THREAD

**Issue by uranik777** [royNiladri/js-big-decimal#49]
bigDecimal.round(0.00000001,8) return "0.000000000100000"
bigDecimal.round("1e-8",8) return "0.000000000100000"
although it should return "0.00000001"

it looks like a bug in the code

error in string:
number = '0.' + (new Array(-exponent + 1)).join('0') + mantisa;
fix:
number = '0.' + (new Array(-exponent - 1)).join('0') + mantisa;

**Comment by royNiladri** [comment]
The fix is a little more elaborate than recommended, but thanks for pointing in the general direction where the issue was present.

**Comment by royNiladri** [comment]
Fixed and generated patched version [v1.3.4](https://www.npmjs.com/package/js-big-decimal). Added test cases in the pipeline:
```
it("should round('1e-8',10) = 0.00000001", function() {
        expect(roundOff(0.00000001,10)).toBe('0.0000000100')
    })
it('should transform 2e-2 to 0.02', function () {
                var n = new bigDecimal('2e-2');
                expect(n.getValue()).toBe('0.02');
    });
```
